article/app/pages/ArticleHtmlPage.scala

```diff
@@ -15,14 +14,8 @@ object ArticleHtmlPage extends HtmlPage[ArticlePage] {
 
     val (header, content): (Html, Html) = page.article match {
       case article if article.isPhotoEssay => (photoEssayHeader(), immersiveBody())
-      case article if article.isImmersive && experiments.ActiveExperiments.isParticipating(experiments.Garnett) => (
-          immersiveGarnettHeader(), immersiveGarnettBody()
-      )
-      case article if article.isImmersive => (immersiveHeader(), immersiveBody())
-      case _ if !request.isAmp && experiments.ActiveExperiments.isParticipating(experiments.Garnett) => (
-        guardianHeaderHtml(), articleBodyGarnett(page)
-      )
-      case _ => (guardianHeaderHtml(), articleBody(page))
+      case article if article.isImmersive => (immersiveGarnettHeader(), immersiveGarnettBody())
+      case _ => (guardianHeaderHtml(), articleBodyGarnett(page))
```

Comment:
We've removed the `!request.isAmp` condition from the final case here, are we using `articleBodyGarnett(page)` for Amp pages too?

Reply:
AMP pages should be decided here: https://github.com/guardian/frontend/blob/master/article/app/controllers/ArticleController.scala#L114

Reply:
Just tested in dev after rebasing and this is the case. Thanks for asking @GHaberis ! I will merge after teamcity passes